capture-traffic/capture-from-ios.md

```diff
@@ -12,70 +12,96 @@ previous_url: /knowledge-base/configure-ios, /get-started/mobile-traffic/configu
+
+1. Start Fiddler Everywhere. 
+
+1. Open the **Home** pane.
+
+1. Open the **Remote Devices** screen.
+
+1. Open the **iOS** tutorial and follow the instructions.
+
+The tutorial requires administrative privileges for the current user (on the iOS device) to install Fiddler's Certificate Authority (CA)and to set manual proxy. Once the steps are executed, you can immediately capture HTTPS traffic from the remote iOS device.
 
-1. Enable the remote connections of Fiddler Everywhere through **Settings** > **Connections** > **Allow remote computers to connect**.
+
+## Capturing Remote iOS Traffic (Manual Setup)
+
+### Configuring Fiddler Everywhere Host
+
+1. Enable the remote connections of Fiddler Everywhere through **Settings** > **Connections** > **Allow remote devices to connect**. Combine it with **Keep it ON after app restart** if you want to persist the setting for subsequent application startups.
 
 1. Check the local IP address of the host where the Fiddler Everywhere application runs. You can use [the connection status on the lower right-hand side]({%slug connections-section%}) to obtain the host IP address. Alternatively, you can get the IP address while using  **ipconfig**/ **ifconfig** (depending on the OS). For demonstration purposes, let's assume that the local IP of the Fiddler Everywhere machine is **192.168.0.101**.
 
-1. Ensure that the Fiddler CA is installed and the [**Capture HTTPS traffic** option]({%slug decrypt-https-traffic%}) is enabled.
+>tip: If you only need to capture remote traffic, then there is no need to install the Fiddler CA on the Fiddler host machine. You only need to download and install the Fiddler CA on the remote device. Once the proxy configuration is in place, Fiddler Everywhere will automatically capture and decrypt the remote HTTPS traffic.
 
-## Configuring iOS Devices
+### Configuring the Fiddler proxy on iOS Device
 
 Refer to the following steps to configure real iOS devices to work alongside a Fiddler Everywhere host. For more information on working with iOS simulators, refer to the section about [configuring iOS simulators](#configure-the-ios-simulator). Real iOS devices and iOS simulators must be discoverable on the same local network.
 
+1. Install the Fiddler's Certificate Authority (CA) on the iOS device.
+
+    1. Open a browser on the iOS device and type the `http://<fiddler-host-IP>:8866` echo service address of Fiddler Everywhere. Follow the Fiddler root certificate link to **download** the Fiddler CA.
+
+        >tip Instead of the special Fiddler echo service address, you can use the Fiddler host IP as follows: `http://<fiddler-host-IP>:8866`.
+
+    1. On your iOS device, open **General** > **Profile Downloaded** and install the downloaded Fiddler CA.
+
+    1. (iOS 10.3+) Go to **Settings** > **General** > **About** > **Certificate Trust Settings** and enable full trust for the **Fiddler Root Certificate Authority** CA.
```

Comment:
do you need CA at the end here? It sounds like "full trust for the **Fiddler Root Certificate Authority** Certificate Authority"